cilium-cli/connectivity/builder/pod_to_pod_encryption_v2.go

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package builder
+
+import (
+	_ "embed"
+
+	"github.com/cilium/cilium/cilium-cli/connectivity/check"
+	"github.com/cilium/cilium/cilium-cli/connectivity/tests"
+	"github.com/cilium/cilium/cilium-cli/utils/features"
+	"github.com/cilium/cilium/pkg/versioncheck"
+)
+
+//go:embed manifests/client-egress-l7-http-from-any.yaml
+var clientsEgressL7HTTPFromAnyPolicyYAMLV2 string
```

Comment:
Do we expect this to differ from `clientsEgressL7HTTPFromAnyPolicyYAML` taken from the older `pod_to_pod_encryption.go`?

Reply:
I added this here to keep everything "self contained" and not have to go look at another file to understand whats going on. 

If you feel strongly that it shouldn't be duplicated I can remove tho.

Reply:
Updated to remove duplicate variable